Rakefile

```diff
@@ -1,37 +1,3 @@
 require 'puppetlabs_spec_helper/rake_tasks'
-require 'puppet-lint/tasks/puppet-lint'
```

Comment:
Will this need to be re-added? I have seen it removed from most modules that has tasks implemented.

Reply:
Good catch, added it back in to be safe 👍